plugins/devtools/README.md

```diff
@@ -240,10 +240,11 @@ To use the permission framework to secure the DevTools route you'll want to do t
    ```
 
 2. Then open the `packages/app/src/App.tsx` file
-3. The add this import after all the existing import statements:
+3. The add these imports after all the existing import statements:
 
    ```ts
    import { devToolsAdministerPermission } from '@backstage/plugin-devtools-common';
+   import { RequirePermission } from '@backstage/plugin-permission-react';
```

Comment:
We might want to adjust this as this import has been part of the `npx @backstage/create-app@latest` for the last 10 months which means most newer instances will have it already: https://github.com/backstage/backstage/blob/23347b795f8915970597b4da2f0d3c52618c1e28/packages/create-app/templates/default-app/packages/app/src/App.tsx#L34 

Would it make sense to include a note instead saying that you "might" need to add this? Sorry, the exact wording isn't coming to me right now